components/ui/button.tsx

```diff
@@ -1,6 +1,6 @@
-import * as React from "react"
 import { Slot } from "@radix-ui/react-slot"
 import { cva, type VariantProps } from "class-variance-authority"
+import * as React from "react"
```

Comment:
po co ten import

Reply:
to jest ten shadcnowy component
<img width="906" alt="image" src="https://github.com/user-attachments/assets/de7eb183-3937-44d8-a2f2-1fa167a98585">

Reply:
używane jeszcze w interfejsie wyzej

Reply:
okej to zostaw